test/LongFile.lean

```diff
@@ -20,49 +20,61 @@ The current value of 1500 does not exceed the allowed bound.
 Please, remove the `set_option linter.style.longFile 1500`.
 -/
 #guard_msgs in
--- Do not allow setting a "small" `longFile` linter option
+-- Do not allow setting a `longFile` linter option if the file does not exceed the `defValue`
 set_option linter.style.longFile 1500
```

Comment:
What do you think about setting this to the default value instead? I wouldn't want to change that default and break the test... (similarly, auditing this file for occurrences of the number 1500 seems useful)

Reply:
Can you add another test setting the option to something *smaller* than the default (i.e., 100 lines or so) --- the goal being to test the very first error message. That seems to be missing.

Reply:
> Can you add another test setting the option to something _smaller_ than the default (i.e., 100 lines or so) --- the goal being to test the very first error message. That seems to be missing.

The very first test of the file verifies that 1500 is not allowed.  I added a test for 1999 at the end of the file, after setting the default value to 2000, to make sure that the check uses `defValue`, rather than `1500`.  I did not add it next to the one that is already checking this, since it would involve re-checking manually that the claimed error that is not tested is still up to date.

Reply:
By the way, there are three `1500`s in the linter file: two are in doc-strings and one is the default value of the linter.  The rest of the file uses the variables, rather than the number.

Reply:
I found one case clearer with replacing it - but agree on the other ones!